src/core/components/model.jsx

```diff
@@ -176,17 +183,20 @@ class ArrayModel extends Component {
   }
 
   render(){
-    let { required, schema, depth, expandDepth } = this.props
+    let { required, schema, depth, name, expandDepth } = this.props
     let items = schema.get("items")
+    let title = schema.get("title") || name
     let properties = schema.filter( ( v, key) => ["type", "items", "$$ref"].indexOf(key) === -1 )
-
+  
     return <span className="model">
-      <span className="model-title">
-        <span className="model-title__text">{ schema.get("title") }</span>
-      </span>
+      {
+        title && <span className="model-title">
+          <span className="model-title__text">{ title }</span>
+        </span>
+      }
       <Collapse collapsed={ depth > expandDepth } collapsedContent="[...]">
         [
-          <span><Model { ...this.props } schema={ items } required={ false }/></span>
+          <span><Model { ...this.props } name="" schema={ items } required={ false }/></span>
```

Comment:
It's definitely a bit of a hack. Here are screenshots showing the difference with `name=""` and without it.

Without `name=""`
<img width="512" alt="screen shot 2017-06-28 at 6 09 00 pm" src="https://user-images.githubusercontent.com/791222/27665852-0c8ce79e-5c2d-11e7-9bdb-bc2560614eb4.png">

With `name=""`
<img width="1431" alt="screen shot 2017-06-28 at 6 09 23 pm" src="https://user-images.githubusercontent.com/791222/27665855-10c458f6-5c2d-11e7-9c0e-505ef581e46b.png">